src/main/scala/com/timeout/KinesisGraphStage.scala

```diff
@@ -21,11 +21,15 @@ object KinesisGraphStage {
   val ProvisionedThroughputExceededExceptionCode = "ProvisionedThroughputExceededException"
 
   private[KinesisGraphStage] val maxBufferSize = 500 // hard limit imposed by AWS
```

Comment:
We call number of bytes and number of records "size" - that can be confusing, but nothing better has came to my mind yet.